pkg/grafana/types.go

```diff
@@ -44,9 +44,12 @@ func (d Datasource) buildJSONData() map[string]interface{} {
 
 func (d Datasource) buildSecureJSONData(organization Organization) map[string]string {
 	tenantIDs := organization.TenantIDs
-	if d.Type != "loki" {
+	if d.UID == "gs-mimir" {
```

Comment:
This is already defined at https://github.com/giantswarm/observability-operator/blob/5eae1c6b9a2cc782e5562c44aefb94ff6700cf49/pkg/grafana/grafana.go#L44C1-L44C25

Please use a same variable in both places

Reply:
I assume you want me to extract it into a shared constant?

Reply:
It's fixed now